addons/l10n_in/models/account_invoice.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = "account.invoice"
+
+
+    @api.depends('amount_total')
+    def _compute_amount_total_words(self):
+        for invoice in self:
+            invoice.amount_total_words = invoice.currency_id.amount_to_text(invoice.amount_total)
```

Comment:
what it invoice.currency_id is not set?
I think (but didn't test it) you need to do:
```python
currency = invoice.currency_id or invoice.company_id.currency_id
invoice.amount_total_words = currency.amount_to_text(invoice.amount_total)
```

Reply:
@qdp-odoo ,
currency_id is required on invoice . 
see : https://github.com/odoo/odoo/blob/master/addons/account/models/account_invoice.py#L318

by default it take journal currency or current user company currency.

Reply:
ok my bad. :-)